e2e-app/src/app/typeahead/validation/typeahead-validation.e2e-spec.ts

```diff
@@ -0,0 +1,26 @@
+import {TypeaheadPage} from '../typeahead.po';
+import {Key, browser} from 'protractor';
+import {expectFocused, openUrl, sendKey} from '../../tools.po';
+
+fdescribe('Typeahead', () => {
```

Comment:
`fdescribe` → please remove focus

My fault, will open a separate PR to break the build in future cases